### PR TITLE
Deploy yvUSDC ETH Put vault

### PR DESCRIPTION
**Pull request by kenchangh**

### CODE REVIEW

File: scripts/deploy/09_yearn_logic.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  const chainId = network.config.chainId;
+
+  const lifecycle = await deployments.get("VaultLifecycle");
+
+  const lifecycleYearn = await deploy("VaultLifecycleYearn", {
+    contract: "VaultLifecycleYearn",
+    from: deployer,
+  });
+
+  await deploy("RibbonThetaVaultYearnLogic", {
```
Comment (fqlx) [inline scripts/deploy/09_yearn_logic.ts:30]: Could you `console.log()` the address?

File: scripts/deploy/09_yearn_logic.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      GAMMA_CONTROLLER[chainId],
+      MARGIN_POOL[chainId],
+      GNOSIS_EASY_AUCTION[chainId],
+      YEARN_REGISTRY_ADDRESS,
+    ],
+    libraries: {
+      VaultLifecycle: lifecycle.address,
+      VaultLifecycleYearn: lifecycleYearn.address,
+    },
+  });
```
Comment (fqlx) [inline scripts/deploy/09_yearn_logic.ts:46]: Also add a verify contract call here?

File: scripts/deploy/10_yvusdc_eth_put_theta_vault.ts [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    initArgs
+  );
+
+  const vault = await deploy("RibbonThetaVaultETHPutYearn", {
+    contract: "AdminUpgradeabilityProxy",
+    from: deployer,
+    args: [logicDeployment.address, admin, initData],
+  });
+
+  console.log(`RibbonThetaVaultETHPutYearn @ ${vault.address}`);
```
Comment (fqlx) [inline scripts/deploy/10_yvusdc_eth_put_theta_vault.ts:81]: Could we verify the contract here?

File: constants/constants.ts [hunk 4]
```diff
@@ -67,7 +67,7 @@ export const SUSHI_OWNER_ADDRESS = {
 };
 
 export const Y_WETH_ADDRESS = "0xa9fE4601811213c340e850ea305481afF02f5b28";
-export const Y_USDC_ADDRESS = "0x5f18C75AbDAe578b483E5F43f12a39cF75b973a9";
+export const YVUSDC_V0_4_3 = "0xa354F35829Ae975e850e23e9615b11Da1B3dC4DE";
```
Comment (fqlx) [inline constants/constants.ts:70]: Why is it version `V0_4_3`?

Comment (kenchangh) [reply constants/constants.ts:71]: Adding the yearn's versioning for yvUSDC. 0.4.3 is the new version.

**Review by fqlx — DISMISSED** [review]

**Review by fqlx — APPROVED** [review]
lgtm

**Pull request merged**